src/components/accessories/table/filter/FilterButton.tsx

```diff
@@ -0,0 +1,234 @@
+import React, {

```

Comment:
All this logic could use some tests + add this to storybook

Reply:
Hello @gasp ! I think we'd better address this in a separate issue.  Please mark  the comment as resolved if you're OK.

Reply:
I am not keen on delaying tests in separate issues especially when it concerns components that are meant to spread all across the app, but that's not my call, you have more experience than I have on this project and I respect your choice and follow your opinion.

Reply:
@SteveGT96 and @gasp how long it would take to add test to this component?

Reply:
In [OH2-301](https://openhospital.atlassian.net/browse/OH2-301) I timeboxed it to 0.5d, it may be a little longer if we face difficulties with storybook that haven't been used for a while. 

I have little velocity measures to make estimates, so this one is certainly as wrong as all the other estimates I have added recently 😅.

As discussed with Steve, this feature is already costly because it's blocking other listings such as [OH2-293](https://openhospital.atlassian.net/browse/OH2-293), [OH2-294](https://openhospital.atlassian.net/browse/OH2-294), [OH2-295](https://openhospital.atlassian.net/browse/OH2-295) so let's prioritize this merge in develop over the tests